src/index.html

```diff
@@ -6,17 +6,402 @@
+                alt="image-product5"
+              />
+            </a>
+
+            <a
+              class="products__image products__image--small"
+              href="#index"
+            >
+              <img
+                class="products__photo"
+                src="./images/Image-product6.png"
+                alt="image-product6"
+              />
+            </a>
+          </div>
+
+          <h3 class="products__items">Accessories</h3>
+        </article>
+
+        <button class="products__button">View all</button>
+      </section>
+
+      <section class="about main__section">
+        <div class="about__image-wrap">
+          <img
+            class="about__image"
+            src="./images/Image-layout.png"
+            alt="image-layout"
+          />
+        </div>
+
+        <div class="about__content">
+          <h2 class="about__tittle">About us</h2>
+          <p class="about__text">
+            We’re a London-based tech company on a mission to remove barriers
+            between people and technology.
+            <br />
+            <br />
+            To do this, we’re crafting intuitive, flawlessly connected products
+            that improve our lives without getting in the way.
+            <br />
+            No confusing tech-speak. No silly product names. Just artistry,
+            passion and trust. And products we’re proud to share with our
+            friends and family. Simple.
+            <br />
+            <br />
+            Because beautiful tech shouldn’t be complicated.
+          </p>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <h2 class="footer__tittle">Contact us</h2>
+
+      <div class="footer__grid">
+        <div class="contact footer__section">
+          <form
+            class="contact__form"
+            action="index.html"
+            method="post"
+            onsubmit="this.reset(); return false;"
+          >
+            <div class="contact__field">
+              <input
+                required
+                placeholder="Name"
+                class="contact__input"
+                type="text"
+              />
+            </div>
+
+            <div class="contact__field">
+              <input
+                required
+                placeholder="Email"
+                class="contact__input"
+                type="email"
+              />
+            </div>
+
+            <div class="contact__field">
+              <textarea
+                required
+                placeholder="Message"
+                class="contact__area"
+                cols="30"
+                rows="10"
+              ></textarea>
+            </div>
+
+            <button class="contact__button">Send</button>
+          </form>
+        </div>
+
+        <div class="footer__section">
+          <div class="footer__sections">
+            <p class="footer__label">Phone</p>
+            <p class="footer__value">+1 234 5555-55-55</p>
```

Comment:
You should highlight contact info on hover [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Change%20text%20color%20on%20hover%20for%20phone%2C%20email%2C%20and%20address)

Also, Phone, and email should be links to start a call or write an email [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Make%20sure%20phone%20icons%20and%20phone%20numbers%20are%20real%20links%20to%20start%20a%20call)